util/chplenv/chpl_cpu.py

```diff
@@ -500,16 +500,25 @@ def get_module_lcd_cpu(platform_val, cpu):
     else:
         return 'none'
 
+@memoize
+def target_is_prgenv():
```

Comment:
Should this be in `compiler_utils`?

Reply:
it's only used in chpl_cpu so I was viewing it as a local helper function.
I can move it there if you think that would be worthwhile.